config/telegramBotHandler.js

```diff
@@ -36,86 +36,86 @@ class TelegramBotHandler {
         });
 
         // Handle the /new_lottery command
-        this.bot.onText(/\/new_lottery/, async (msg) => {
```

Comment:
When user enter the telegram, userID has to be catched.

Reply:
I fixed.